cdap-ui/app/directives/my-pipeline-summary/my-pipeline-summary.html

```diff
@@ -35,8 +35,13 @@
         <td>{{MyPipelineSummaryCtrl.totalRunsCount}}</td>
         <td>{{MyPipelineSummaryCtrl.avgRunTime}}</td>
         <td>
-          {{ MyPipelineSummaryCtrl.nextRunTime | amDateFormat:'h:mm:ss A'}}
-          <small> {{ MyPipelineSummaryCtrl.nextRunTime | amDateFormat:'MM/DD/YYYY'}}</small>
+          <span ng-if="MyPipelineSummaryCtrl.nextRunTime !== '--'">
```

Comment:
instead of comparing against an empty string (which can be ambiguous) or '--' (which was chosen just arbitrarily for this case), reverting back to comparing against 'N/A' as that seems to be a common test in the app